cli/commands.js

```diff
@@ -57,22 +62,61 @@ const defaultCommands = {
     heroku: {
         desc: 'Deploy the current application to Heroku',
     },
-    'import-jdl': {
+    info: {
+        desc: 'Display information about your current project and system',
+    },
+    jdl: {
+        alias: 'import-jdl',
         argument: ['jdlFiles...'],
         cliOnly: true,
+        options: [
+            {
+                option: '--skip-install',
+                desc: 'Do not automatically install dependencies',
+                default: false,
+            },
+            {
+                option: '--interactive',
+                desc: 'Run generation in series so that questions can be interacted with',
+                default: false,
+            },
+            {
+                option: '--db <value>',
+                desc: 'Provide DB option for the application when using skip-server flag',
+            },
+            {
+                option: '--json-only',
+                desc: 'Generate only the JSON files and skip entity regeneration',
+                default: false,
+            },
+            {
+                option: '--ignore-application',
+                desc: 'Ignores application generation',
+                default: false,
+            },
+            {
+                option: '--ignore-deployments',
+                desc: 'Ignores deployments generation',
+                default: false,
+            },
+            {
+                option: '--skip-ui-grouping',
+                desc: 'Disable the UI grouping behavior for entity client side code',
+                default: false,
+            },
+            {
+                option: '--skip-db-changelog',
+                desc: 'Disable generation of database changelogs',
+                default: false,
+            },
+            {
+                option: '--inline <value>',
+                desc: 'Pass JDL content inline. Argument can be skipped when passing this',
+            },
```

Comment:
Instead of passing an inline JDL that can potentially override the sample entirely, this should be used to customize the pulled sample.
@mshima @deepu105 

And there shouldn't be any need for `--db` or `--blueprint/s`

Reply:
This is a kickstart feature, the user don't need to know the jdl structure.